### ISSUE TITLE
How to track visualization on BDD

### ISSUE THREAD

**Issue by monster519** [SysCV/qdtrack#57]
I tried python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [--out ${RESULT_FILE}] [--eval ${EVAL_METRICS}] [--show] [--show-dir ${SHOW_DIR}] [--cfg-options]
but  I didn't get the results of track visualization on BDD

**Comment by XiaLiPKU** [comment]
Currently, you can resort to this process:
1. Convert the prediction into BDD100K format: https://github.com/SysCV/qdtrack/blob/master/docs/GET_STARTED.md#conversion-to-the-scalabelbdd100k-format
2. Visualize the prediction through bdd100k tools: https://doc.bdd100k.com/usage.html#understanding-the-data

**Comment by taheranjary** [comment]
Hi there. I ran your pre-trained BDD100K model on the tracking set and saved the results to `result.pkl`. Then I ran the command `python tools/to_bdd100k.py ./configs/bdd100k/qdtrack-frcnn_r50_fpn_12e_bdd100k.py --res results.pkl --task box_track --bdd-dir conversions --nproc 1 --coco-file COCO_PRED_FILE`. This script complains saying the file should be a list. If I convert it to a list, consequent scripts complain that the object has no `.items()`. So what should the result file be, a dictionary or a list?
